etl/steps/data/garden/who/2024-01-03/gho.excluded_countries.json

```diff
@@ -1,7 +1,19 @@
 [
-    "Africa (2) (UN)",
-    "Americas (UN)",
-    "Asia (142) (UN)",
-    "Europe (150) (UN)",
-    "Oceania (9) (UN)"
+  "Africa (2) (UN)",
+  "Americas (UN)",
+  "Asia (142) (UN)",
+  "Europe (150) (UN)",
+  "Oceania (9) (UN)",
+  "Africa (2) (UN)",
+  "Asia (142) (UN)",
+  "Africa (2)",
+  "Asia (142)",
+  "Least Developed Countries (LDCs)",
+  "Northern America (21)",
+  "Northern America",
+  "GlobalGlobal",
```

Comment:
Isn't GlobalGlobal both in the mapping and in excluded countries?

Reply:
Ah yes it was! I've removed it from here now.